src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -751,4 +705,34 @@ private void assertActionCommentIsAbsentInLoan(JsonObject loan) {
     assertThat("'actionComment' property should be absent in loan",
       loan.getString(ACTION_COMMENT_KEY), nullValue());
   }
+
+  private void verifyRenewedLoan(IndividualResource smallAngryPlanet,
+    IndividualResource jessica, JsonObject renewedLoan) {
+    assertThat("user ID should match barcode",
+      renewedLoan.getString("userId"), is(jessica.getId().toString()));
+
+    assertThat("item ID should match barcode",
+      renewedLoan.getString("itemId"), is(smallAngryPlanet.getId().toString()));
+
+    assertThat("status should be open",
+      renewedLoan.getJsonObject("status").getString("name"), is("Open"));
+
+    assertThat("action should be renewed",
+      renewedLoan.getString("action"), is(RENEWED_THROUGH_OVERRIDE));
+
+    assertThat("'actionComment' field should contain comment specified for override",
+      renewedLoan.getString(ACTION_COMMENT_KEY), is(OVERRIDE_COMMENT));
+  }
+
+  private void declareItemLost(JsonObject loanJson) {
+    final UUID loanId = UUID.fromString(loanJson.getString("id"));
+    final String comment = "testing";
+    final DateTime dateTime = DateTime.now(DateTimeZone.UTC);
+
+    final DeclareItemLostRequestBuilder builder = new DeclareItemLostRequestBuilder()
+      .forLoanId(loanId).on(dateTime)
+      .withComment(comment);
```

Comment:
Could this logic be hidden in the fixture method for declaring an item lost?

Reply:
Done